.github/workflows/acceptance.yml

```diff
@@ -0,0 +1,51 @@
+name: Acceptance Test CI
```

Comment:
"Acceptance Test" should be good enough. The word will show up in the Action page. Action is already CI.